code/modules/mob/dead/observer/login.dm

```diff
@@ -1,30 +1,34 @@
 /mob/dead/observer/Login()
-	. = ..()
```

Comment:
This entire file has the wrong indentation, I can revert if you'd prefer but while im here i might as well fix it.

Reply:
This is a gray area, really.
The PR is not massive enough for the code cleanup to impact it negatively IMO, so keep it.